.gitignore

```diff
@@ -3,3 +3,4 @@
 ################################################################################
 
 /.vs
+ModUtil/**
```

Comment:
why was this needed? not an issue more just curious

Reply:
It's not needed, just an artifact of me floundering around before I learned what submodules on GitHub actually are and how they worked.

> we should probably redo this as a git submodule the same way we have many of the others, given that RunHistoryWipe has its own repo.

Sounds good, I will work on getting that set up.